client/src/components/PetStats/__tests__/PetName.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Component from './../PetName';
+describe('PetName', ()=>{
+    it('renders without error', ()=>{
+        const div = document.createElement('div');
+        ReactDOM.render(<Component/>, div);
+        ReactDOM.unmountComponentAtNode(div);
```

Comment:
Where are you actually running the test? There should be an expect() or something that reports whether or not that div was rendered or not.

Reply:
Oh and this should be done in any place you have this render test